WBooks/Library/LibraryViewController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  LibraryViewController.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 8/21/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class LibraryViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    struct Constants {
+        static let backgroundBlueColor = UIColor(hex: "EAF6FA")
+        static let tablePositionY: CGFloat = 40
+        static let defultEdgeValue: CGFloat = 0
+    }
+    
+    // MARK: - Properties
+    
+    private let _libraryViewModel: LibraryViewModel!
+    private let _booksViewcontroller: BooksTableViewController
```

Comment:
You could do this a `private lazy var _booksViewcontroller = BooksTableViewController(booksViewModel: self._libraryViewModel.createBooksViewModel())`
So you avoid lines of code in the init, and you avoid creating the controller until necessary.